pages/cultural-dimensions.py

```diff
@@ -5,12 +5,27 @@
 
 dash.register_page(
     __name__,
-    description="This app uses tabs to highlight cultural dimensions on a bar chart.",
+    description="This app uses tabs to highlight cultural dimensions on a bar chart. It has a top-bottom layout and a regular-callback.",
 )
 
 filename = __name__.split("pages.")[1]
 
-# make a mini layout with the links to the docs that would go under each app
-# links = html.Div([])
+# any notes will be displayed below the code-and-show page in a dcc.Markdown component
+notes = """
+#### Dash Components in App:
+- [Dropdown](https://dash.plotly.com/dash-core-components/dropdown)
 
-layout = example_app(filename)
+#### Plotly Documentation:
+- [Bar Charts](https://plotly.com/python/bar-charts/)
+
+#### Community Components:
+
+Dash Bootstrap Components 
+- [Layout](https://dash-bootstrap-components.opensource.faculty.ai/docs/components/layout/ "This component was made by the community and not officially maintained by Plotly.")
```

Comment:
@tolgahancepel 
please take out the layout link and replace it with the 2 links that refer to the tabs and label dbc components used in the app:
[Label](https://dash-bootstrap-components.opensource.faculty.ai/docs/components/input/ "This component was made by the community and not officially maintained by Plotly.")
[Tabs](https://dash-bootstrap-components.opensource.faculty.ai/docs/components/tabs/ "This component was made by the community and not officially maintained by Plotly.")